src/lib.rs

```diff
@@ -327,17 +328,52 @@ impl<'a> arbitrary::Arbitrary<'a> for SmolStr {
     }
 }
 
-const INLINE_CAP: usize = 22;
+const INLINE_CAP: usize = 23;
 const N_NEWLINES: usize = 32;
 const N_SPACES: usize = 128;
 const WS: &str =
     "\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n                                                                                                                                ";
 
+#[derive(Clone, Copy, Debug)]
+#[repr(u8)]
+enum InlineSize {
```

Comment:
Using macro or proc macro is possible -- however 24 isn't too many

Reply:
Variants are guaranteed to pick the previous ones value + 1, so just specifying the first (for good measure) is enough